go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/minio/minio-go/v7
 
-go 1.17
+go 1.21.3
```

Comment:
```suggestion
go 1.19
```

Reply:
1.19 suffers from the same vulnerabilities unfortunately.  I saw that you had some CICD to be backward compatible with 1.19 which might be a bigger change.  I'm happy to accept the changes but they won't solve the issue.